site/docs/voice/rateLimits.mdx

```diff
@@ -21,4 +21,10 @@ If a rate limit is hit during a `<Transfer>`, the error will be sent to `transfe
 
 As an alternative to hard rate limiting for outbound calls, you may have your account manager enable outbound call queueing. When enabled, your outbound calls will be queued and initiated at a specific dequeueing rate, enabling your application to "fire and forget" when creating calls. Your queue size is limited to a maximum of 5 minutes worth of outbound calls at your dequeing rate. Queued calls may not be modified until they are dequeued and placed, but may be removed from your queue on demand.
 
+:::note
+
+The ordering of calls placed in your queue is not guaranteed. Calls submitted to your queue may be placed in any order, regardless of the order they were submitted.
```

Comment:
I wouldn't say _any_ order. It's correctish. As written, it kind of implies that it's random which it isn't. Maybe we should say that they will be sent approximately in order, but that the order isn't strictly guaranteed?